internal/service/rds/cluster.go

```diff
@@ -236,6 +236,27 @@ func ResourceCluster() *schema.Resource {
 				},
 			},
 
+			"serverlessv2_scaling_configuration": {
```

Comment:
I'm not so familiar with golang but could this not be wrapped up in scaling configuration by making `max_capacity` and `min_capacity` either `TypeInt` or `TypeFloat`. It seems like this is an unnecessary duplication but again I'm not knowledgable enough to say whether it is or not for certain

Reply:
While this would technically be possible, it’s just the representation of how AWS solves it within their SDK and API.

Reply:
We prefer to stick to the representation in the underlying AWS API(s), so `serverlessv2_scaling_configuration` is correct here.